linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/sh
+
+# Capture CLI arguments
+cmd=$1
+db_username=$2
+db_password=$3
+
+# Start docker
+# Make sure you understand the double pipe operator
+sudo systemctl status docker || systemctl start docker
+
+# Check container status (try the following cmds on terminal)
+docker container inspect jrvs-psql
+container_status=$?
```

Comment:
something like `docker container ls` with `name=____` would be preferable to inspect, which produces a huge json dump to the terminal. You can then also combine these lines into something like
`container_status = $(yourcommand)`

Reply:
Also, the comment is generally not needed here. It's pretty clear what these lines do since you've named your variable appropriately